config/configgrpc/configgrpc.go

```diff
@@ -165,6 +165,10 @@ type GRPCServerSettings struct {
 
 	// Auth for this receiver
 	Auth *configauth.Authentication `mapstructure:"auth,omitempty"`
+
+	// MuxListener is the multiplexing listener for the Endpoint of NetAddr. Through multiplexing, we can
+	// support many sockets on the same port with the same listener.
```

Comment:
Some minor doc clarification:
```suggestion
	// MuxListener is a multiplexing listener for the Endpoint of NetAddr. Through multiplexing, 
	// a single listener can support multiple sockets on the same port.
```